packages/semantic-ui/package.json

```diff
@@ -39,33 +39,34 @@
     "semantic-ui-react": "^1.3.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.17",
+    "@babel/core": "^7.23.7",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
-    "@babel/preset-env": "^7.22.15",
-    "@babel/preset-react": "^7.22.15",
-    "@babel/preset-typescript": "^7.22.15",
+    "@babel/preset-env": "^7.23.7",
+    "@babel/preset-react": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
     "@rjsf/core": "^5.15.1",
     "@rjsf/snapshot-tests": "^5.15.1",
     "@rjsf/utils": "^5.15.1",
     "@rjsf/validator-ajv8": "^5.15.1",
-    "@types/jest": "^29.5.7",
-    "@types/lodash": "^4.14.198",
-    "@types/react": "^17.0.65",
-    "@types/react-dom": "^17.0.20",
-    "@types/react-test-renderer": "^17.0.3",
+    "@types/jest": "^29.5.11",
+    "@types/lodash": "^4.14.202",
+    "@types/react": "^17.0.74",
+    "@types/react-dom": "^17.0.25",
+    "@types/react-test-renderer": "^17.0.9",
     "atob": "^2.1.2",
-    "babel-jest": "^29.6.4",
+    "babel-jest": "^29.7.0",
     "esbuild": "^0.18.20",
-    "eslint": "^8.48.0",
-    "jest": "^29.6.4",
-    "jest-environment-jsdom": "^29.6.4",
-    "nanoid": "^3.3.6",
+    "eslint": "^8.56.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-environment-jsdom-global": "^4.0.0",
```

Comment:
ditto

Reply:
Since this is in the main `package.json` I can remove this one